app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -70,6 +71,7 @@ object TelemetryWrapper {
         val SHARE_INTENT = "share_intent"
         val REMOVE = "remove"
         val REORDER = "reorder"
+        val RECEIVE = "receive"
```

Comment:
Receive isn't very specific, and it looks like maybe we want to differentiate between an error from the current page or a page resource. I'd actually add two methods, PAGE and RESOURCE, and add a comment about what they are (see the onReceivedSslError method and comment for the two cases).